src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -381,27 +382,22 @@ public TextAssessmentResource(AuthorizationCheckService authCheckService, TextAs
     }
 
     /**
-     * Retrieves all the text submissions that have conflicting feedback with the given feedback id.
+     * GET text-submissions/:submissionId/feedback/:feedbackId/feedback-conflicts : Retrieves all the text submissions that have conflicting feedback with the given feedback id.
      * User needs to be either assessor of the submission (with given feedback id) or an instructor for the exercise to check the conflicts.
      *
      * @param submissionId - id of the submission with the feedback that has conflicts
      * @param feedbackId - id of the feedback that has conflicts
      * @return - Set of text submissions
      */
-    @GetMapping("/submission/{submissionId}/feedback/{feedbackId}/feedback-conflicts")
+    @GetMapping("/text-submissions/{submissionId}/feedback/{feedbackId}/feedback-conflicts")
```

Comment:
* remove "/"
* use plural

When I review the implementation of this method, I think we have a problem here. The authorization checks are done based on the `submissionId`, however this is not relevant for the objects that are returned. Someone could invoke this method with an arbitrary submission and feedback id.